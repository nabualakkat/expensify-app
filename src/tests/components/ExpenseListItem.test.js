import React from 'react'
import {shallow} from 'enzyme'
import ExpenseListItem from '../../components/ExpenseListItem'



test('should render Expense List Item', () => {
    const wrapper = shallow(<ExpenseListItem id = '1' description = 'Water' amount={300000} createdAt={231412341243}/>)
    expect(wrapper).toMatchSnapshot()
})