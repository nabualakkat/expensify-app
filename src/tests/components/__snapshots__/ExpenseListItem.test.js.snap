// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense List Item 1`] = `
<Link
  className="list-item"
  to="/edit/1"
>
  <div>
    <h3
      className="list-item__title"
    >
      Water
    </h3>
    <span
      className="list-item__subtitle"
    >
      May 2nd, 1977
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $3,000.00
  </h3>
</Link>
`;
